modules/rosa-sdpolicy-am-regions-az.adoc

```diff
@@ -60,6 +60,7 @@ endif::rosa-with-hcp[]
 ifndef::rosa-with-hcp[]
 * eu-west-3 (Paris)
 endif::rosa-with-hcp[]
+* eu-central-2 (Zurich)
```

Comment:
A couple of things to check:
1. It looks like you conditionalized this new region for ROSA with HCP, however, the `rosa list regions` command on the CLI shows that this region is also available for Classic. Should it be included there too?
2. eu-central-2 might be an AWS opt-in zone based on [this page from AWS](https://docs.aws.amazon.com/controltower/latest/userguide/opt-in-region-considerations.html). I'm not sure, but you might want to check it out.
3. Don't forget to add your new region (for HCP and Classic) to the ROSA "What's New" page.

Reply:
@pawelpaszki and @sdminonne - can you let me know if this region is available on Classic and if it's opt in? I think the former is true, but I do not know about the latter.